src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -404,6 +404,12 @@ public virtual bool RunInMemory
         [IndexUpdateType(IndexUpdateType.Reset)]
         [ConfigurationEntry(Constants.Configuration.Indexes.IndexingStaticSearchEngineType, ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public SearchEngineType StaticIndexingEngineType { get; protected set; }
+
+        [Description("Corax index compression max documents used for dictionary creation.")]
+        [DefaultValue(100000)]
+        [IndexUpdateType(IndexUpdateType.Reset)]
+        [ConfigurationEntry(Constants.Configuration.Indexes.IndexingCompressionMaxDocuments, ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
Why not put `"Indexing.Corax.Compression.MaxDocuments"` directly? I don't see any usage of `IndexingCompressionMaxDocuments` const.

I wonder if `"Indexing.Corax.DocumentsLimitForCompressionDictionaryCreation"` or something similar wouldn't be more descriptive.

Reply:
I like that name.